package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "yarn": "^1.22.19"
```

Comment:
`yarn` also shouldn't be in our dependencies

Reply:
I dont remember doing what caused this